roles/obsd_httpd/files/staging_env.conf

```
@@ -9,23 +9,23 @@ events {
 error_log  logs/error.log;
 error_log  logs/error.log  notice;
 error_log  logs/error.log  info;
-error_log  syslog:server=unix:/staging/log,severity=notice;
+error_log  syslog:server=unix:/dev/log,severity=notice;
```

Should this still be staging?
```suggestion
error_log  syslog:server=unix:/staging/log,severity=notice;
```

---

I am very sorry for these errors, I have corrected these and committed the corrections.